app/experimenter/nimbus-ui/src/components/FormRequestReview/index.tsx

```diff
@@ -0,0 +1,89 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useState } from "react";
+import { Alert, Form } from "react-bootstrap";
+
+type FormRequestReviewProps = {
+  isLoading: boolean;
+  submitError: string | null;
+  onSubmit: () => void;
+};
+
+const FormRequestReview = ({
+  isLoading,
+  submitError,
+  onSubmit,
+}: FormRequestReviewProps) => {
+  const [checkedBoxes, setCheckedBoxes] = useState<{
+    onboarding: boolean;
+    risks: boolean;
+  }>({
+    onboarding: false,
+    risks: false,
+  });
+  const allBoxesChecked = Object.values(checkedBoxes).every((v) => v === true);
+
+  return (
+    <Form className="py-2">
+      {submitError && (
+        <Alert data-testid="submit-error" variant="warning">
+          {submitError}
+        </Alert>
+      )}
+
+      <Form.Row className="my-3">
+        <Form.Group className="w-100" controlId="checkbox-onboarding">
+          <Form.Check
+            type="checkbox"
+            label="I have gone through the experiment onboarding program"
+            onClick={(
+              event: React.MouseEvent<HTMLInputElement, globalThis.MouseEvent>,
+            ) => {
+              const checked = (event.target as HTMLInputElement).checked;
+              setCheckedBoxes((existing) => {
+                existing.onboarding = checked;
+                return { ...existing };
+              });
+            }}
```

Comment:
Super nitty code golfing, but possible improvements:
```
const handleConfirmChange =
  (name: string) =>
  (event: ev: React.ChangeEvent<HTMLInputElement>) =>
  setCheckedBoxes((existing) => ({ ...existing, [name]: ev.target.checked })
```
Then you can use `onChange={handleConfirmChange('onboarding')}` and `onChange={handleConfirmChange('risks')}`

Reply:
Nice. I also updated these checkboxes to render based on an array of label strings.